README.rst

```diff
@@ -121,6 +121,11 @@ To learn more, see:
     SIGNALFX_TRACING_URL=tracing endpoint [ default: https://ingest.signalfx.com/v1/trace ]
     
 
+4. (Optional) Set environment variable to set `environment` root span tag
+
+.. code:: bash
+    SIGNALFX_ENVIRONMENT=production
```

Comment:
Do other signalfx libraries also have a `SIGNALFX_ENVIRONMENT` env var option? I've generally seen other libraries take a comma separated list of arbitary tags and expect users to supply environment as part of that.